api/passwordless/routing.go

```diff
@@ -0,0 +1,30 @@
+package passwordless
+
+import (
+	"github.com/keratin/authn-server/api"
+	"github.com/keratin/authn-server/lib/route"
+)
+
+func PublicRoutes(app *api.App) []*route.HandledRoute {
+	originSecurity := route.OriginSecurity(app.Config.ApplicationDomains)
+
+	routes := []*route.HandledRoute{
+		route.Post("/passwordless/login").
```

Comment:
i'd like to merge these new endpoints into the `/sessions` api directory:

* `.Get("/sessions/token")`
* `.Post("/sessions/token")`

Reply:
do you think we should also change the function names to `getToken` and` postToken` or we keep the current names?

Reply:
It'd be nice to keep parity between func names and routes.

What do you think about `getToken` and `postToken`? It's the best I could come up with on the spot, but I'm open to other ideas if you think it should be clearer.

Reply:
maybe `getSessionToken` and `postSessionToken` to use the same pattern of `.Get("/session/refresh")` with `getSessionRefresh`.